src/k8s/pkg/k8sd/api/cluster.go

```diff
@@ -40,7 +41,7 @@ func (e *Endpoints) getClusterStatus(s state.State, r *http.Request) response.Re
 		return response.InternalError(fmt.Errorf("failed to check if cluster has ready nodes: %w", err))
 	}
 
-	var statuses map[string]types.FeatureStatus
+	var statuses map[features.FeatureName]types.FeatureStatus
```

Comment:
For the `FeatureStatus` I was following the proposal (since it was already accepted), but for this one it just felt right to put it here. but I get what you're saying. I'd say that now I prefer the `FeatureStatus` to move into the `k8sd/feature` package, what do you think @neoaggelos?

Reply:
the `type FeatureName string` should be in `pkg/k8sd/types`, the constants should be in `pkg/k8sd/features`